demo_nodes_cpp/src/services/add_two_ints_client.cpp

```diff
@@ -45,7 +55,17 @@ int main(int argc, char ** argv)
 
   auto node = rclcpp::Node::make_shared("add_two_ints_client");
 
-  auto client = node->create_client<example_interfaces::srv::AddTwoInts>("add_two_ints");
+  if (cli_option_exist(argv, argv+argc, "-h")) {
```

Comment:
If these are C functions, `cli_option_exist()` and others, then I think you ought to namespace them, e.g. `utilities_cli_option_exist()`.